extensions/google-search/package.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "google-search",
+  "title": "Google Search",
+  "description": "Google search with autosuggestions",
+  "icon": "command-icon.png",
+  "author": "mblode",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Google or type a URL",
+      "subtitle": "Google",
```

```suggestion
      "title": "Google Search",
```
Normally we use the subtitle, but in this case it doesn't make sense

---

Fixed 👍